src/main/java/com/tagtraum/perf/gcviewer/imp/DataReaderShenandoah.java

```diff
@@ -88,7 +94,10 @@ public GCModel read() throws IOException {
         model.setFormat(GCModel.Format.RED_HAT_SHENANDOAH_GC);
 
         Stream<String> lines = new BufferedReader(in).lines();
```

Comment:
No need to wrap "in" in a BufferedReader. It is a LineNumberReader, which is a subclass of BufferedReader.